axelor-admin/src/main/java/com/axelor/apps/base/service/ObjectDataAnonymizeServiceImpl.java

```diff
@@ -55,6 +55,9 @@ public void anonymize(ObjectDataConfig objectDataConfig, Long recordId) throws A
       String rootModel = objectDataConfig.getModelSelect();
 
       for (DataConfigLine line : objectDataConfig.getDataConfigLineList()) {
+        String path =
+            line.getTypeSelect() == 0 ? line.getMetaFieldPath().getName() : line.getPath();
```

Comment:
Use constant for typeSelect like 'TYPE_PATH' instead of static value  '0', replace it everywhere in service.